src/Extensions/Shopware/DataGenerator/Services/LoadDataInfile.php

```diff
@@ -83,7 +83,7 @@ class LoadDataInfile
         's_articles_categories' => ['articleID', 'categoryID'],
         's_media' => ['id', 'albumID', 'name', 'path', 'type', 'extension', 'created'],
         's_articles_img' => ['articleID', 'img', 'main', 'extension', 'media_id'],
-        's_articles_attributes' => ['articleID', 'articledetailsID'],
+        's_articles_attributes' => ['id', 'articledetailsID'],
```

Comment:
You can just remove articleID :thinking: . Its since 5.2 null

Reply:
correct. so remove it, and we are happy to merge your fix :blush: